src/GlobalContext/index.js

```diff
@@ -0,0 +1,679 @@
+import { createContext, useState, useEffect, useContext } from "react";
+import Cookies from "js-cookie";
+import { StateManager } from "../App";
+import { useNavigate } from "react-router-dom";
+
+export const GlobalProvider = createContext();
+export const GlobalContext = (props) => {
```

Comment:
The one that's wrapping the children / consumer should be called the provider,
```suggestion
export const GlobalContext = createContext();
export const GlobalProvider = (props) => {
```